src/backend/utils/misc/guc.c

```diff
@@ -3771,7 +3789,7 @@ static struct config_enum ConfigureNamesEnum[] =
 			NULL
 		},
 		&wal_level,
-		WAL_LEVEL_MINIMAL, wal_level_options,
+		WAL_LEVEL_LOGICAL, wal_level_options,
```

Comment:
Let's keep this to `WAL_LEVEL_MINIMAL` and note in the docs that users need to change it to `WAL_LEVEL_LOGICAL` if they want to use triggers. If you're going to do that in a separate pass, that's fine too.